readme.md

```diff
@@ -33,10 +33,34 @@ Homebridge is setup via `config.json` file sitting in the `~/.homebridge/` direc
   {
     "platform": "Ecobee 3 Sensors",
     "name": "Ecobee",
-    "exclude_thermostat": false
+    "exclude_thermostat": false,
+    "apiKey":"YOUR API KEY"
   }
 ]
 ```
+To obtain your own api key:
+#### 1. Go to [Ecobee login page](https://www.ecobee.com/home/ecobeeLogin.jsp)
+
+<img src="images/ecobee-auth-1.png" width="45%">
+
+#### 2. Login with your username and password
+You need to register your thermostat first, so do this if you haven't yet.
+
+<img src="images/ecobee-auth-2.png" width="45%">
+
+#### 3. Select _DEVLOPER_ from the menu on the top right.
+<img src="images/devloper.png" width="45%">
+
+#### 4. Click _CREATE NEW_.
+
+#### 5. Fill in Application Name and Application Summary.
+<img src="images/app.png" width="45%">
+
+#### 6. Click _Authorization Method _. and select ecobee PIN
+<img src="images/pin.png" width="45%">
+
+#### 7. Click _CREATE _. and copy your API key.
+
```

Comment:
I love the screenshot style docs! Only thing I'm not sure about is the placement. Are you sure about showing the developer API key as one of the first things in the `readme.md` file? Maybe I'm wrong but, personally, I consider this a feature for more advanced users. I know you probably don't feel like it since you wrote it, but I'm afraid that people who see the repository for the very first time can be easily scared if we show them too much.

Since the API key isn't needed to get the plugin started for the first time, maybe we can place it little bit further down. How do you feel about moving this under a new FAQ question: _Can I use my own Ecobee developer API key?_ Or starting a wiki page for the repo?